src/Map/src/Bridge/Google/assets/src/map_controller.ts

```diff
@@ -146,24 +170,31 @@ export default class extends AbstractMapController<
             ...rawOptions,
         });
 
-        if (definition.opened) {
-            infoWindow.open({
-                map: this.map,
-                shouldFocus: false,
-                anchor: marker,
+        if (element instanceof google.maps.marker.AdvancedMarkerElement) {
+            element.addListener('click', () => {
+                if (definition.autoClose) this.closeInfoWindowsExcept(infoWindow);
```

Comment:
Not a fan of one-lined condition + code, as it can easily introduce mistakes, please prefer keeping this form:
```suggestion
                if (definition.autoClose) {
                    this.closeInfoWindowsExcept(infoWindow);
                }
```

Note: I will see if Biome can prevent this